clang/include/clang/Basic/AttrDocs.td

```diff
@@ -2667,30 +2671,6 @@ In OpenCL C, this attribute is available with the GNU spelling
 
   __kernel __attribute__((reqd_work_group_size(8, 16, 32))) void test() {}
 
-  }];
-}
-
-def WorkGroupSizeHintAttrDocs : Documentation {
-  let Category = DocCatFunction;
-  let Content = [{
-Applies to a device function/lambda function. Hint to the compiler on the work-
-group size most likely to be used when launching the kernel at runtime. Each
-argument must be a nonzero integral constant expression whose value is
-unsigned. The number of dimensional values defined provide additional
-information to the compiler on the dimensionality most likely to be used when
-launching the kernel at runtime.
-
-The GNU spelling is deprecated in SYCL mode.
-
-.. code-block:: c++
-
-  [[sycl::work_group_size_hint(4, 4, 4)]] void foo() {}
-
-  class Foo {
-  public:
-    [[sycl::work_group_size_hint(2, 2, 2)]] void operator()() const {}
-  };
-
```

Comment:
wrong merge conflicts happened. This patch fixes it

Reply:
Why are these docs moving around?

Reply:
The line: The arguments to ``reqd_work_group_size`` are ordered based on which index...should be added with ReqdWorkGroupSizeAttrDocs but due to wrong merge, this was added with WorkGroupSizeHintAttrDocs. This is NFC change, @AaronBallman, if you prefer, i could do this in a separate PR.

Reply:
Separating into an NFC would likely make it more clear for code archeology purposes.

Reply:
Thanks @AaronBallman. Done